<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuSolver</class>
 <widget class="QWidget" name="SudokuSolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="SudokuTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>541</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>9</number>
   </property>
   <property name="columnCount">
    <number>9</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>10</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>10</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="8">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>54</red>
       <green>238</green>
       <blue>100</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="1" column="6">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="7">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="8">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>57</red>
       <green>208</green>
       <blue>22</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="7">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="5">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="6">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="8">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="7">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="6">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="7">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="Line" name="vline1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>41</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="vline2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>41</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="hline1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="hline2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sudokuSolver">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sudoku Solver</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="solveButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
